packages/node/src/indexer/store.service.ts

```diff
@@ -249,6 +279,13 @@ export class StoreService {
           throw new Error('Relation type is not supported');
       }
     }
+    foreignKeyMap.forEach((keys, tableName) => {
```

Comment:
Should this be under a `this.historical` check?

Reply:
not really, it's not a no-op if historical=false